janitor/functions.py

```diff
@@ -777,11 +777,25 @@ def label_encode(
         or tuple) of column names.
     :returns: A pandas DataFrame.
     """
-    warnings.warn("label_encode will be deprecated in a 1.x release")
+    warnings.warn(
+        "label_encode will be deprecated in a 1.x release. \
+        Use factorize_columns instead"
```

Comment:
```suggestion
        Please use factorize_columns instead"
```

As a Canadian, I insist we be Canadian-polite :joy_cat: